src/main/java/nl/thehyve/ocdu/models/OcDefinitions/SiteDefinition.java

```diff
@@ -20,7 +20,12 @@
 
     private boolean isGenderRequired = true;
 
-    private boolean isBirthdateRequired = true;
+    /*
+     * 1. yes, required
+     * 2. only year of birth
```

Comment:
yes, actually there is an integer [1,3] indicating 1.full date, 2.only year, and 3.not used, I have changed the code correspondingly, please check.